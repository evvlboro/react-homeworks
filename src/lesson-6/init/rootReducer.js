// Core
import { combineReducers } from 'redux';

// Reducers
import { studentReducer as student } from '../bus/student/reducer';

export const rootReducer = combineReducers({
  student
});
