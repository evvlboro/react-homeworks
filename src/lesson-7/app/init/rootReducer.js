import { combineReducers } from 'redux';

import { peopleReducer as people } from '../bus/people/reduser';
import { filmsReducer as films } from '../bus/films/reduser';
import { filmReducer as film } from '../bus/film/reduser';
import { personReducer as person } from '../bus/person/reduser';

export const rootReducer = combineReducers({
  people,
  person,
  films,
  film
})
