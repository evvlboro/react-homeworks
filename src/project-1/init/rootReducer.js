import { combineReducers } from 'redux';

import { forecastReducer as forecast } from '../bus/forecast/reducer';
import { uiReducer as ui } from '../bus/ui/reducer';

export const rootReducer = combineReducers({
  forecast,
  ui
});
