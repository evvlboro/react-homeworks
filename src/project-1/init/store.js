import {createStore, applyMiddleware} from 'redux';

//Core
import {rootReducer} from './rootReducer';
import {composeEnhancers, middleware} from './middleware';

export const store = createStore(rootReducer, composeEnhancers(applyMiddleware(...middleware)));