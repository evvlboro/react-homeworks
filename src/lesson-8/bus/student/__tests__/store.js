import { store } from '../../../init/store';

it('test store', () => {
  expect(store.getState()).toMatchSnapshot();
});
