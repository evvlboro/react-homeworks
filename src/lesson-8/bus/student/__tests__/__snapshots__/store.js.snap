// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test store 1`] = `
Object {
  "student": Object {
    "data": Object {
      "age": "",
      "email": "",
      "firstName": "",
      "sex": "",
      "speciality": "",
      "surname": "",
    },
  },
}
`;
